appserver/admingui/payara-console-extras/src/main/resources/asadminRecorder/asadminRecorderTabs.jsf

```diff
@@ -44,16 +44,9 @@
     />
     
     <!command
-        setSessionAttribute(key="serverInstTabs" value="asadminRecorderConfig");
-        gf.redirect(page="#{request.contextPath}/payaraExtras/asadminRecorder/asadminRecorderServer.jsf?target=server&tabSetName=serverInstTabs");
+        setSessionAttribute(key="domainTabs" value="asadminRecorderConfig");
+        gf.redirect(page="#{request.contextPath}/payaraExtras/asadminRecorder/asadminRecorderServer.jsf?target=server&tabSetName=domainTabs");
     />
-
-    <sun:tab id="asadminRecorderConfig" immediate="$boolean{true}" text="$resource{i18nar.asadminRecorder.configuration}" toolTip="$resource{i18nar.asadminRecorder.configurationTip}" >
```

Comment:
Can you remove the _asadminRecorder.configuration_ and _asadminRecorder.configurationTip_ Strings from the _Strings.properties_ file please? They're no longer needed.